notebooks/sen2_blade_csv_eda.clj

```diff
@@ -115,3 +119,16 @@
 ;; Note: OK if not a unique key without `requests-table-id`,
 (sen2-blade-csv-eda/report-composite-keys sen2-blade-csv-ds-map)
 
+
+
+
+^{::clerk/visibility {:code :hide, :result :hide}}
+(comment ;; clerk build
+  (let [in-path            (str "notebooks/" (clojure.string/replace (str *ns*) #"\.|-" {"." "/" "-" "_"}) ".clj")
+        out-path           (str wk-dir (clojure.string/replace (str *ns*) #"^.*\." "") ".html")]
+    (clerk/build! {:paths    [in-path]
+                   :ssr      true
+                   :bundle   true
+                   :out-path "."})
+    (.renameTo (io/file "./index.html") (io/file out-path)))
+  )
```

Comment:
I want to use the adroddiad stuff for this, but I don't want to bring in adroddiad as a dependency

Reply:
I also didn't want to introduce adroddiad dependencies, so duplicated some of the required age/ncy stuff in a local ns `witan.sen2.ncy` (introduced in PR#4).